pep-0007.txt

```diff
@@ -50,8 +50,8 @@ C dialect
   Future C99 features may be added to this list in the future
   depending on compiler support (mostly significantly MSVC).
 
-* Don't use GCC extensions (e.g. don't write multi-line strings
-  without trailing backslashes).
+* Don't use compiler extensions, such as those of GCC or MSVC
```

Comment:
```suggestion
* Don't use compiler-specific extensions, such as those of GCC or MSVC
```

I don't know if "compiler extensions" is a term of art. If not, "-specific" sounds better to me.

A

Reply:
I was going to go with this to begin with, but it didn't make the final edit for some reason. Thanks.